### ISSUE TITLE
Windows 10 error

### ISSUE THREAD

**Issue by tdivito** [KKBOX/CompassApp#312]
I just upgraded to Windows 10, and started getting a bunch of errors.  Compass will compile for 15-20 minutes, then just error out with the below messages

![image](https://cloud.githubusercontent.com/assets/13768412/9229126/9b0a139a-40e9-11e5-9599-641d082f1f18.png)

Thanks for taking a look

**Comment by hlb** [comment]
@tdivito does the project work before win10 upgrade?

**Comment by tdivito** [comment]
Yes.   I was working on this project for a few weeks, and it started to error the morning after the windows 10 upgrade

Thanks

**Comment by elochai90** [comment]
I had this problem, too (win10). I found out, that only if the path to the watched folder has less than 6 parent folders the compiling works. So "C:/[folder1/folder2/folder3/folder4/folder5/]my_watched_folder" works (for me). But I need my folder on the 9th level (). Any ideas?

**Comment by thwbn** [comment]
Hi,
if it is not too late: What do you think about a junction? Create a folder on first level and link this to your x-leveled folder you need to work with. So all your your files you create in the first level folder are in reality on the x level folder and compass App should think to be in first level....

https://msdn.microsoft.com/de-de/library/windows/desktop/aa365006%28v=vs.85%29.aspx

greetings from Bonn

Thorsten
